src/reducers/comments.spec.tsx

```diff
@@ -19,35 +25,80 @@ describe(__filename, () => {
 
   describe('beginComment', () => {
     it('begins a comment by key', () => {
-      const fileName = 'manifest.json';
-      const line = 123;
-      const versionId = 1;
+      const params = { versionId: 1, fileName: 'manifest.json', line: 123 };
 
-      const state = reducer(
-        undefined,
-        actions.beginComment({ versionId, fileName, line }),
+      const state = reducer(undefined, actions.beginComment(params));
+
+      expect(state.byKey[createCommentKey(params)]).toEqual(
+        createCommentInfo({ beginNewComment: true }),
```

Comment:
Would there be any value in switching this to use `expect.objectContaining({ beginNewComment: true })`, rather than `createCommentInfo({ beginNewComment: true })`? I suppose the latter is less implementation-specific, so it seems better in that case, but I notice we do often use `expect.objectContaining` in the tests.